databricks-sdk-java/src/main/java/com/databricks/sdk/core/UserAgent.java

```diff
@@ -13,7 +13,7 @@ public class UserAgent {
   // TODO: check if reading from
   // /META-INF/maven/com.databricks/databrics-sdk-java/pom.properties
   // or getClass().getPackage().getImplementationVersion() is enough.
```

Comment:
We can probably add this TODO in next fixit, it would be nice to decouple it with release process. Currently I assume this has to be done manually?

Reply:
Nope, it is automated!